.travis.yml

```diff
@@ -35,11 +35,13 @@ before_install:
   - git clone -b release-1-6-924 --depth 1 https://github.com/bioperl/bioperl-live.git
 
 install:
-  - cpanm -v --installdeps --notest .
+  - cpanm -v --installdeps --notest . --with-all-features
   - cpanm -n Devel::Cover::Report::Coveralls
   - cpanm -n DBD::SQLite
   - cp travisci/MultiTestDB.conf.travisci.mysql  modules/t/MultiTestDB.conf.mysql
   - cp travisci/MultiTestDB.conf.travisci.SQLite modules/t/MultiTestDB.conf.SQLite
+  - cp travisci/testdb.conf.travisci.mysql  testdb.conf.mysql
+  - cp travisci/testdb.conf.travisci.SQLite testdb.conf.SQLite
   - mysql -u root -h localhost -e 'GRANT ALL PRIVILEGES ON *.* TO "travis"@"%"'
```

Comment:
This would replace master branch's .travis.yml with an older version

Reply:
Rebased against current _master_ again to incorporate changes recently made there.

Reply:
Unfortunately, still bringing in changes that aren't in _master_, even after the rebase.

Reply:
Well, those two lines _are_ supposed to be added, for both GitLab-CI and Travis, by this PR; otherwise the xref test suite does not run. Or have I misunderstood?

Reply:
Ah, I'm the one who misunderstood. They're fine, then.